configs/benchmarks/classification/imagenet/vit-base-p16_ft-8xb128-coslr-100e-rpe_in1k.py

```diff
@@ -5,18 +5,16 @@
 ]
 
 # dataset
-preprocess_cfg = dict(
+data_preprocessor = dict(
+    num_classes=1000,
     # RGB format normalization parameters
```

Comment:
Fix the mean and std, and add comments to help the author using the correct setting.